src/input/Input.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import style from "input/Input.module.css";
+
+const Input = ({
+                   disabled,
+                   textLabel,
+                   caption,
+                   error,
+                   value,
+                   onChangeHandler,
+                   placeholderText
+               }
+) => {
+
+    return (
+        <>
+            <div className={style.wrapper}>
+                {textLabel && <div className={style.textLabel}>{textLabel}</div>}
+            </div>
+            <input value={value}
+                   onChange={onChangeHandler}
+                   className={error ? `${style.input} ${style.error}` : style.input}
+                   disabled={disabled}
+                   placeholder={placeholderText}
+            />
+            {!error ? <div className={style.caption}>{caption}</div> :
```

Comment:
caption can also be empty. better to check if caption exists before rendering it